css-four-column-layout/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Four Column Layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+
+  <div class="container soft-gray-bg text-barlow">
+
+    <header class="row justify-center">
+      <div class="column-full">
+        <h2 class="text-extra-bold">How it Works</h2>
+      </div>
+    </header>
+
+    <main class="row justify-center text-center">
+      <div class="column-fifth">
```

Comment:
make sure you have an explicit width set for your column classes!

Reply:
Ahh - I totally forgot! I got lucky with the initial width of my container... Will fix now.